esbuild-runner.mjs

```diff
@@ -0,0 +1,217 @@
+    },
+    "module": {
+        type: "boolean",
+        shortKey: true,
+        propertySyntax: false,
+        default: false
+    }
+});
+const params = parseArgs();
+async function build(params) {
+    const swcConfig = {
+        "jsc": {
+            "externalHelpers": true,
+            "transform": {
+                "optimizer": {
+                  "simplify": true
+                }
+            },
+            "minify": {
+                "compress": {
+                    "arguments": true,
+                    "booleans_as_integers": true,
+                    "ecma": 5,
+                    "arrows": true,
+                    "booleans": true,
+                    "collapse_vars": true,
+                    "comparisons": true,
+                    "conditionals": true,
+                    "dead_code": true,
+                    "defaults": true,
+                    "directives": true,
+                    "drop_console": false,
+                    "drop_debugger": true,
+                    "evaluate": true,
+                    "hoist_funs": false,
+                    "hoist_props": true,
+                    "hoist_vars": false,
+                    "if_return": true,
+                    "inline": true,
+                    "join_vars": true,
+                    "keep_classnames": false,
+                    "keep_fargs": false,
+                    "keep_infinity": false,
+                    "loops": true,
+                    "negate_iife": true,
+                    "passes": 0,
+                    "properties": true,
+                    "pure_getters": "",
+                    "reduce_funcs": false,
+                    "reduce_vars": true,
+                    "sequences": true,
+                    "side_effects": true,
+                    "switches": true,
+                    "top_retain": "",
+                    "toplevel": true,
+                    "typeofs": true,
+                    "unsafe": false,
+                    "unsafe_arrows": false,
+                    "unsafe_comps": false,
+                    "unsafe_Function": false,
+                    "unsafe_math": false,
+                    "unsafe_symbols": false,
+                    "unsafe_methods": false,
+                    "unsafe_proto": false,
+                    "unsafe_regexp": false,
+                    "unsafe_undefined": false,
+                    "unused": true
+                },
+                "mangle": false
+            }
+        },
+        "rootMode": "root",
+        "root": Path.dirname(params.options.input??params.options.args[0]),
+        "sourceMaps": params.options.sourcemap??false,
+        //"inlineSourcesContent": true,
+        "isModule": true,
+        "minify": params.options.minify??false
+    };
+    let esbuildTarget;
+    {
+        //These are the targets for a compatibility build that should work with all browsers in use today, even in embedded devices.
+        const compatTargets = {
```

Comment:
Maybe we could put these into package.json and retrieve them from there, so it will be easier to maintain, as we'll probably forget about this file if we don't have to touch it very often.

Reply:
> Maybe we could put these into package.json and retrieve them from there, so it will be easier to maintain, as we'll probably forget about this file if we don't have to touch it very often.

done.